src/client/package.json

```diff
@@ -12,6 +12,7 @@
     "core-js": "^3.6.5",
     "vue": "^2.6.11",
     "vue-router": "^3.5.1",
+    "vue-session": "^1.0.0",
```

Comment:
i remember Jessy saying something about not needing "vue-session" but im not entirely sure

Reply:
Let's wait for Jessy to confirm

Reply:
oh keep vue-session. I was referring to flask-sessions